runtime/port/common/j9ri.h

```diff
@@ -60,14 +59,18 @@ typedef struct riControlBlock {
    uint32_t branchOnCondAsOtherTypeBrranches:1;
    uint32_t RIEmit:1;
    uint32_t TXAbort:1;
-   uint32_t pad3:2;
+
+   uint32_t guardedStorage:1;
+   uint32_t pad3:1;
```

Comment:
Could we use a better name for the paddings? It would be much easier to make sure the offsets are correct if for example `pad3` was renamed to be something like `padAtBit15`.

Reply:
Sure, with the first padding, `pad29`, is that name clear since it is a 29 bit padding at the start of word 6 or should we think of something more descriptive?

Reply:
Let's make the naming consistent. It shouldn't matter whether the pad is at the start on in the middle. The name should be descriptive enough to tell you where the padding is. The bit field will always tell how big the padding is.

Reply:
424f9aa